sencha-workspace/packages/slate-cbl/src/view/teacher/demonstration/EditWindow.js

```diff
@@ -270,5 +276,38 @@ Ext.define('Slate.cbl.view.teacher.demonstration.EditWindow', {
         } else {
             me.on('render', _fireEvent, me, { single: true });
         }
+    },
+    
+    applyDeleting: function(deleting) {
+        var me = this,
+            toolbar,
+            _doApplyDeleting = function () {
+                //disable form fields
+                Ext.Array.each(me.query('[disableOnDelete]'), function(component) {
+                    component.setDisabled(deleting);
+                });
+                
+                //disable Competencies
+                Ext.Array.each(me.query('[maskOnDelete]'), function(component) {
+                    if(component.rendered) {
+                        if(deleting) { 
+                            component.mask();
+                        } else {
+                            component.unmask();
+                        }
+
+                    }
+                });
+                
+                toolbar = me.getComponent('deletionToolbar');
+
+                toolbar.setHidden(!deleting);
+            };
+        
+        if(me.rendered) {
+            _doApplyDeleting();
```

Comment:
i'm not seeing the problem with any of these indentations, but there should be a space between the if and the (